modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,209 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple clusters, thus helping to avoid a single point of failure. 
+
+[NOTE]
+====
+{product-title} vSphere topology awareness does not support multiple vCenter datacenters.
+====
+
+This is accomplished by defining zone and region categories in vCenter, and then assigning these categories to different failure domains, such as a compute cluster, host, or datacenter, by creating tags for these zone and region categories. After you have created the appropriate categories, and assigned tags to vCenter objects, you can create additional machinesets that create virtual machines (VMs) that are responsible for scheduling pods in those failure domains.
```

Comment:
nit: We should drop datacenter from here too, since we don't support individual datacenter as a failure-domain in <=4.12